src/tasks/copy-template-files.ts

```diff
@@ -290,34 +269,31 @@ export async function copyTemplateFiles(options: Options, templateDir: string, t
   const tmpDir = path.join(targetDir, EXTERNAL_EXTENSION_TMP_FOLDER);
 
   // 1. Copy base template to target directory
-  await copyBaseFiles(options, basePath, targetDir);
-
-  // 2. Add "parent" extensions (set via config.json#extend field)
-  options.extensions = expandExtensions(options);
```

Comment:
We no more need this since we are not allowing [extending extension](https://arc.net/l/quote/pcfbofja) anymore